legal_references.tsv

```diff
@@ -3,7 +3,7 @@ adopsjonsloven	lov/1986-02-28-8
 advokatforskriften	forskrift/1996-12-20-1161
 afp-tilskottsforskriften	forskrift/2010-11-30-1496
 agenturloven|agentloven|agl	lov/1992-06-19-56
-aksjeloven|asl	lov/1997-06-13-44
+aksjeloven|asl	lov/1997-06-13-44|lov/1976-06-04-59|lov/1957-07-06-4
```

Comment:
lov/1957-07-06-4 er aksjeselskapsloven, ikke aksjeloven, såvidt jeg ser på lovdata. Tror kanskje den hører hjemme på sin egen linje, med mindre det også er riktig å referere til den som aksjeloven. Hør mer Rolf.

Reply:
![image](https://user-images.githubusercontent.com/12566527/74225964-f3236580-4cbb-11ea-8359-96112749b733.png)
Den ble referert som "aksjeloven", men skal dobbeltsjekke med Rolf.

Reply:
Rolf vil at den skal legges til slik at vi får lenke til riktig URL, men det er riktig det du sier. Forfatteren er upresis her.

Reply:
Ok, det viktigste er jo at vi oppdager så mange referanser som mulig og genererer riktigst mulig lenker for dem, så da må vi jo ta høyde for at forfattere tar seg noen frihet i hvordan de refererer til visse verk.